content/_includes/page/meta.njk

```diff
@@ -37,7 +37,7 @@
         {{ post.pubdate(
           start=page.date,
           end=end,
-          format='range' if oss or client or event else 'long'
+          format='range' if oss or client or ('_talk' in tags) else 'long'
```

Comment:
`('_talk' in tags)` might be risky if `tags` is undefined?

Reply:
Hm, maybe? It doesn't seem to cause any build errors... I think 11ty ensures that `tags` always exists, it's just empty on some pages.

Reply:
oh also: things only show up in these lists if they have the relevant tags 😀